app/templates/home.hbs

```diff
@@ -4,8 +4,12 @@
   sortableColumns=sortableColumns
   data=model.books
   total=model.total
+  orderBy=orderBy
```

Comment:
The idea of a component was to encapsulate the component user from the orderby, skip acrobats.